src/main/java/uk/ac/ebi/embl/api/validation/GenomeUtils.java

```diff
@@ -0,0 +1,27 @@
+package uk.ac.ebi.embl.api.validation;
+
+import uk.ac.ebi.embl.api.entry.AssemblySequenceInfo;
+
+import java.util.*;
+
+public class GenomeUtils {
+
+    public final static Long COVID_19_OUTBREAK_GENOME_MAX_SIZE = 31000L; //bp
+
+    public static Long calculateGenomeSize(Map<String, AssemblySequenceInfo> sequenceInfo, Set<String> agpPlacedComponents) throws ValidationEngineException {
+        long genomeSize = 0;
+        for (Map.Entry<String, AssemblySequenceInfo> entry : sequenceInfo.entrySet()) {
+            AssemblySequenceInfo info = entry.getValue();
+
+            if (info.getAssemblyLevel() == 0 || info.getAssemblyLevel() == 1) {
+                if (!agpPlacedComponents.contains(entry.getKey())) {
```

Comment:
- We might want to uppercase here as well for safety reasons or have unit tests that also check that we can match placed sequences case-insensitively.